src/core/spark.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  BUILD_BRANCH,
+  BUILD_PULL_REQUESTS,
+  BUILD_REPOSITORY_URL,
+  BUILD_SHA1,
+  BUILD_TAG,
+} from "secrets";
+
+import { FIREBASE } from "./firebase";
+
+const logout = () =>
+  void FIREBASE.auth.signOut().then(console.log).catch(console.error);
```

Comment:
weird console usages if you ask me

Reply:
Yeah, I meant to replace it with more elaborate one, like the capture of error

Reply:
Done 4425bc5465061b654e0802f90fd88d3e4d895b70